server/controllers/api/videos.js

```diff
@@ -643,5 +643,7 @@ function addVideoToBlacklist (req, res, next) {
       logger.error('Errors when blacklisting video ', { error: err })
       return next(err)
     }
+
+    return res.type('json').status(204).end()
```

Comment:
I have left `return res.type('json').status(204).end()` because I get an error on the client when blacklisting a video otherwise. To be more precise, everything works fine, the video _is_ blacklisted but the notification shows an error status.